src/app/RankStore.js

```diff
@@ -0,0 +1,12 @@
+const maxRankingSize = 3;
+
+export const getRanking = (mode) =>
+  JSON.parse(localStorage.getItem(mode)) || [];
+
+export const rankPlayer = (nickname, score, mode) => {
```

Comment:
You could use destructuring in parameter:
`export const rankPlayer = ({nickname, score, mode})`
or take that info from a summary object
`export const rankPlayer = (gameSummary)`

Reply:
Do you want me to do that or it's just a suggestion?